inst/lib/bsw/dist/minty/_bootswatch.scss

```diff
@@ -25,57 +25,6 @@ $web-font-path: "https://fonts.googleapis.com/css2?family=Montserrat:wght@400;50
 
 .btn {
   font-family: $headings-font-family;
-
-  &,
-  &:hover {
-    color: $white;
-  }
-
-  &-light,
-  &-light:hover {
-    color: $gray-700;
-  }
-
-  &-link,
-  &-link:hover {
-    color: $primary;
-  }
-
-  &-link.disabled:hover {
-    color: $gray-600;
-  }
-
-  &-outline-primary {
-    color: $primary;
-  }
-
-  &-outline-secondary {
-    color: $secondary;
-  }
-
```

Comment:
We need to keep the outline classes in. Ex: `btn btn-outline-primary` in https://bootswatch.com/minty/

Reply:
Turns out these rules are actually redundant -- the weirdness I was seeing it actually a result of https://github.com/rstudio/bslib/issues/250 (I'll fix that here as well)